vspace/src/main/java/edu/asu/diging/vspace/core/factory/IExhibitionAboutPageFactory.java

```diff
@@ -0,0 +1,15 @@
+package edu.asu.diging.vspace.core.factory;
+
+import java.util.List;
+
+import edu.asu.diging.vspace.core.model.ILocalizedText;
+import edu.asu.diging.vspace.core.model.impl.ExhibitionAboutPage;
+
+public interface IExhibitionAboutPageFactory {
+    
+    /**
+     * Creates Exhibition About Page
+     */
+    ExhibitionAboutPage createAboutPageForm(String title, String aboutPageText,  List<ILocalizedText> localizedTitles, List<ILocalizedText> localizedDescriptions);
```

Comment:
does this create a form? the return type doesn't indicate that, but the name does.